test-suite/generated-src/java/com/dropbox/djinni/test/Conflict.java

```diff
@@ -11,9 +11,9 @@
  * Test for conflict of method name with an interface name.
  * See the comments about scopeSymbols in CppMarshal.scala for more info.
  */
-public abstract class Conflict {
+public interface Conflict {
 
-    private static final class CppProxy extends Conflict
+    static final class CppProxy implements Conflict
```

Comment:
Shouldn't this be private?

Reply:
Java interfaces can't have private members. I read that they'll be able to in Java 9.